web/src/features/prompts/components/NewPromptForm/index.tsx

```diff
@@ -0,0 +1,367 @@
+                  control={form.control}
+                  name="chatPrompt"
+                  render={({ field }) => (
+                    <>
+                      <PromptChatMessages {...field} />
+                      <FormMessage />
+                    </>
+                  )}
+                />
+              </TabsContent>
+            </Tabs>
+          </FormItem>
+          <p className="text-sm text-gray-500">
+            You can use <code className="text-xs">{"{{variable}}"}</code> to
+            insert variables into your prompt.
+            {currentExtractedVariables.length > 0
+              ? " The following variables are available:"
+              : ""}
+          </p>
+          <div className="flex min-h-6 flex-wrap gap-2">
+            {currentExtractedVariables.map((variable) => (
+              <Badge key={variable} variant="outline">
+                {variable}
+              </Badge>
+            ))}
+          </div>
+        </>
+
+        {/* Prompt Config field */}
+        <FormField
+          control={form.control}
+          name="config"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Config</FormLabel>
+              <JsonView
+                src={jsonSchema.parse(JSON.parse(field.value))}
+                onEdit={(edit) => {
+                  field.onChange(JSON.stringify(edit.src));
+                }}
+                editable
+                className="rounded-md border border-gray-200 p-2 text-sm"
+              />
+              <FormDescription>
+                Track configs for LLM API calls such as function definitions or
+                LLM parameters.
+              </FormDescription>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+
+        {/* Activate prompt field */}
+        <FormField
+          control={form.control}
+          name="isActive"
+          render={({ field }) => (
+            <FormItem className="flex flex-row items-center space-x-3 space-y-0 rounded-md border p-4">
+              <FormControl>
+                <Checkbox
+                  checked={field.value}
+                  onCheckedChange={field.onChange}
+                />
+              </FormControl>
+              <div className="space-y-1 leading-none">
+                <FormLabel>Activate prompt</FormLabel>
+              </div>
+              {currentIsActive ? (
+                <div className="text-xs text-gray-500">
+                  Activating the prompt will make it available to the SDKs
+                  immediately.
+                </div>
+              ) : null}
+            </FormItem>
+          )}
+        />
+        <Button
+          type="submit"
+          loading={createPromptMutation.isLoading}
+          className="w-full"
+          disabled={Boolean(form.formState.errors.name?.message)} // Disable button if prompt name already exists. Check is dynamic and not part of zod schema
```

Comment:
When creating a new version of a prompt and adjusting the prompt text, i am not able to save this. I run the server without experimential features.